Always_green_CI.md

```diff
@@ -0,0 +1,56 @@
+# Introduction
+
+We have occasionally seen that the CI fails due to flaky tests or jobs, environment issues and more. This presents a challenge for new contributors as they need to spend time figuring out issues that are not related to their changes in the code. It also hampers the confidence the community has in the CI as a simple yes/no answer and instead talks about must vs optional tests. We believe that this will become an increasing bottleneck if the kata community is successful in kicking off the new confidential containers project which will bring in additional participating companies and individual contributors. Given this, we established a process and policy that should allow the project to maintain a "Always Green CI" aiming to improve the results of Kata Containers CI shown in GitHub.
+
+# Concepts
+
+In this section is discussed the main pillars that sustain the "Always Green CI" initiative.
+
+## Baseline jobs
+
+There was created multiple jobs (called "baselines") which run on daily basis for the different test suites and configuration/HW which are identical to what will run on each pull request (PR). The key point is that this CI runs on a branch with no additional patches/noise added by anyone so if there is a failure it’s much easier to pinpoint it. The baseline jobs indicate the overall status of the Kata Containers project and we pursue to keep them "always green". They are used as a base for comparing with the CI jobs ran on each pull request (PR): if a given CI job fails but its corresponding on baseline is "green" then it is likely the code change breaking tests. 
+
+Each and every baseline CI job **must have** owners monitoring it’s status and acting if something goes wrong. Policies and a process to add/remove jobs to/from the baseline were defined.
+
+Find the complete list of baseline jobs [here](http://jenkins.katacontainers.io/view/Daily%20baseline/).
```

Comment:
Aside (random thoughts - nothing to do with this PR really ;): I do wonder if we should rename this site to `ci.katacontainers.io` (or even `status.katacontainers.io`) since:

- We might use something other than Jenkins one day.
- It's more discoverable / "guessable"

Reply:
IMHO it's a good idea, I've always avoided referring the technology  on names. I've seen the growth of a new phenomenon: projects written in rust being prefixed with 'rs'. Why??! I understand when the project is a re-write of a famous C library and so on; but for a brand new project it makes no sense :) 

`ci.katacontainers.io` looks good to me BTW.

Reply:
And as if by magic http://ci.katacontainers.io is now active and redirects to http://jenkins.katacontainers.io! :smile:

/cc @chavafg, @GabyCT, @kata-containers/ci.